File: Java/CodingBat/arrayFront9.java
// http://codingbat.com/prob/p186031

public boolean arrayFront9(int[] nums) {
  int numberOfIterations = (nums.length <= 4) ? nums.length : 4;
  for (int i = 0; i < numberOfIterations; ++i) {
    if (nums[i] == 9) {
      return true;
    }
  }
  return false;
}

File: Python/CodingBat/array_front9.py
# http://codingbat.com/prob/p110166

def array_front9(nums):
  if len(nums) >= 4:
    for i in range(4):
      if nums[i] == 9:
        return True
  else:
    for num in nums:
      if num == 9:
        return True
        
  return False
